### PR TITLE
Show missing implicitly added members, add some tests

### PR DESCRIPTION
**Pull request by pikinier20**
Examples: https://scala3doc.s3.eu-central-1.amazonaws.com/pr-211/self/scala3doc/tests/implicitConversions/index.html

### CODE REVIEW

File: src/main/scala/tests/implicitConversions/A.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package tests.implicitConversions
+
+implicit val convDtoC: Conversion[ClazzD, ClazzC]
+    = ???
+
+class ClazzA
+{
+    /* Methods inside this comment block should be added by implicit conversion
+    val prop1: String
+        = ???
```
Comment (romanowski) [inline src/main/scala/tests/implicitConversions/A.scala]: Nitpick: We don't need `= ???` with comments :)

File: src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala [hunk 2]
```diff
@@ -204,11 +204,15 @@ trait ClassLikeSupport:
 
     val methodKind = 
       if methodSymbol.isClassConstructor then Kind.Constructor 
-      else if methodSymbol.flags.is(Flags.Implicit) then extractImplicitConversion(method.returnTpt.tpe) match
-        case Some(conversion) if paramLists.size == 0 || (paramLists.size == 1 && paramLists.head.size == 0) => 
-          Kind.Implicit(Kind.Def, Some(conversion))
-        case _ => 
-          Kind.Implicit(Kind.Def, None)
+      else if methodSymbol.flags.is(Flags.Implicit) then
```
Comment (romanowski) [inline src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala:207]: The origin is a bit misleading [here](https://scala3doc.s3.eu-central-1.amazonaws.com/pr-211/self/scala3doc/tests/implicitConversions/-clazz-a/index.html). It should said  that conversion comes from `object A` and coverts to `class B` - so we would need to change  what is stored in `conversion`

File: src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala [hunk 3]
```diff
@@ -19,16 +19,26 @@ class ImplicitMembersExtensionTransformer(ctx: DokkaContext) extends Documentabl
             val companion = c match 
                 case classlike: DClass => ClasslikeExtension.getFrom(classlike).flatMap(_.companion).map(classlikeMap)
                 case _ => None
+
+            val parentCompanions = c.parents.map { p =>
```
Comment (romanowski) [inline src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala]: Nitpick:

```
val allParents =c.parents.flatMap(p => classlikeMap.get(p.dri))
val parentsCompanions = allParents.flatMap{ _ match 
    case cls: DClasslike => ClasslikeExtension.getFrom(cls).flatMap(_.companion).map(classlikeMap)
    case _ => None
}

``

**Review by romanowski — CHANGES_REQUESTED** [review]
We should expand information in `Kind.Implicit`.

Also few small code style suggestions.

File: src/main/scala/tests/implicitConversions/A.scala (hunk 1 above)
Comment (pikinier20) [reply src/main/scala/tests/implicitConversions/A.scala]: removed

File: src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala (hunk 2 above)
Comment (pikinier20) [reply src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala:207]: Extended information stored in ImplicitConversion and ImplictlyAddedBy

File: src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala (hunk 3 above)
Comment (pikinier20) [reply src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala]: Refactored

File: src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala [hunk 4]
```diff
@@ -19,23 +19,33 @@ class ImplicitMembersExtensionTransformer(ctx: DokkaContext) extends Documentabl
             val companion = c match 
                 case classlike: DClass => ClasslikeExtension.getFrom(classlike).flatMap(_.companion).map(classlikeMap)
                 case _ => None
-            
-            val implictSources = outerMembers ++ companion.toSeq // We can expand this on companion object from parents, generic etc.
 
-            val MyDri = c.getDri
+            val allParents = c.parents.flatMap { p =>
```
Comment (romanowski) [inline src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala:23]: Nitpick: it should be oneliner

File: src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala [hunk 5]
```diff
@@ -19,23 +19,33 @@ class ImplicitMembersExtensionTransformer(ctx: DokkaContext) extends Documentabl
...
                 case _ => None
-            
-            val implictSources = outerMembers ++ companion.toSeq // We can expand this on companion object from parents, generic etc.
 
-            val MyDri = c.getDri
+            val allParents = c.parents.flatMap { p => 
+                classlikeMap.get(p.dri)
+            }
+
+            val parentCompanions = allParents.flatMap { _ match
```
Comment (romanowski) [inline src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala:27]: `_ match` can be ommited in such case

File: src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala [hunk 6]
```diff
@@ -19,23 +19,33 @@ class ImplicitMembersExtensionTransformer(ctx: DokkaContext) extends Documentabl
...
+                    case cls: DClasslike => ClasslikeExtension.getFrom(cls).flatMap(_.companion).map(classlikeMap)
+                    case _ => None
+            }     
+
+            val implictSources = outerMembers ++ companion.toSeq ++ parentCompanions // We can expand this on generic etc.
```
Comment (romanowski) [inline src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala:32]: I think we should create an issue and mentioned it here with `// TODO (#issue_no): We can expand this on generic etc.` style of comment

**Review by romanowski — APPROVED** [review]
I've asked to change a comment plus few style-related nitpicks

**Pull request merged**